typescript/web/src/components/labelling-tool/openlayers-map/edit-label-class.tsx

```diff
@@ -75,9 +75,12 @@ export const EditLabelClass = forwardRef<
     [labelClasses, datasetId]
   );
   useHotkeys(
-    keymap.changeClass.key,
+    "*", // We have to manually check if the input corresponds to a change class key because otherwise on AZERTY keyboards we can't change classes when pressing numbers
     (keyboardEvent) => {
-      if (isOpen) {
+      if (
+        keymap.changeClass.key.split(",").includes(keyboardEvent.key) &&
+        isOpen
+      ) {
         // We do not want to interfere with the class menu shortcuts if this modal is closed
         const digit = Number(keyboardEvent.code[5]);
         const indexOfLabelClass = (digit + 9) % 10;
```

Comment:
You did not have to change this but it works on Azerty and Qwerty keyboards ?

Reply:
Yes check the GIF, I change between spanish and french keyboards. How would you have done it without changing this? I had a look at what had been done to support the slash hotkey and applied the same thing here

Reply:
I would have used `keyboardEvent.key` instead of `keyboardEvent.code` , but it's fine if it works like that ✅